content/help/addons/social/index.md

```diff
@@ -1,16 +0,0 @@
----
-title: Social Connect Plugins
-tags:
-- Features
-- Addons
-- Integrations
-category: help
-menu:
-  help:
-    parent: addons
-    identifier: addons-social
----
-
-## Social Connect Plugins
-
-Vanilla Forums integrates with a number of social applications that you can use as SSO solutions. Learn more about configuring these applications to work with your forum in this section. 
```

Comment:
Was this info added anywhere? I don't see an alias either.

Reply:
I would probably keep this page and append all the social integrations inside of it instead of having them as separate pages.

Reply:
Fixed